src/maintainer/adding_pkgs.rst

```diff
@@ -3,15 +3,19 @@
 Contributing packages
 *********************
 
+Process
+===================
 
-To submit a package to the ``conda-forge`` channel, add its ``recipe`` and licence to the ``staged-recipes`` repository and create a pull request. Once the pull request is merged, the package becomes available
-on the ``conda-forge`` channel. Note that contributing a package makes you the ``maintainer`` of that package.
+To submit a package to conda-forge one has to follow the steps:
```

Comment:
"To submit a package to conda-forge, follow these steps:"

It's better to use what we call second person imperative in documentation. Basically, write like you're giving an order (but in a nice way, of course). :)